content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm

```diff
@@ -50,7 +50,7 @@
                     (<a href="#" onclick="openMoreInfoCopyrightWindow('copyright');" title="[$tlang.getString('cright.newwin')]"><span class="skip"> [$tlang.getString('cright.newwin')]</span>$tlang.getString('cright.fairuse')</a>)
                 </small>
             </div>
-            <div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="show"#else class="hide"#end>
+            <div id="copyright_status"#if($model.copyrightStatus == $USE_THIS_COPYRIGHT) class="d-none"#end>
```

Comment:
This doesn't look right. Should it not be != ?

Reply:
Good catch.